rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/utils/ConnectUtil.java

```diff
@@ -154,6 +156,45 @@ public static DefaultMQAdminExt startMQAdminTool(ConnectConfig connectConfig) th
         return defaultMQAdminExt;
     }
 
+    public static void createTopic(ConnectConfig connectConfig, TopicConfig topicConfig) {
+        DefaultMQAdminExt defaultMQAdminExt = null;
+        try {
+            defaultMQAdminExt = startMQAdminTool(connectConfig);
+
+            Set<String> masterSet = CommandUtil.fetchMasterAddrByClusterName(defaultMQAdminExt, connectConfig.getClusterName());
```

Comment:
When a topic is automatically created, is the clusterName required in the connect config, and can it be obtained through the name server?